docs/contributors/readme.md

```diff
@@ -0,0 +1,15 @@
+# Contributors Guide
+
+Welcome to the Gutenberg Project Contributors Guide.
+
+The following guidelines are in place to create consistency across the project and the numerous contributors. See also the [Contributing Documentation](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for technical details around setup, and submitting your contributions.
+
+
+* [Coding Guidelines](../contributors/coding-guidelines.md) outline additional patterns and conventions used in the Gutenberg project.
```

Comment:
These should be either `../../docs/contributors` or I think we're going to try `/docs/contributors` and get the handbook parser updated.

Reply:
Updated to `../../docs`
I think if the parser is updated to use `/docs/` than the links won't work in Github.